voice-assistant/esp32-s3-box-3.yaml

```
@@ -31,17 +43,14 @@ psram:
   speed: 80MHz
 
 external_components:
-  - source: github://pr#5230
-    components: esp_adf
+  - source: github://pr#5827
+    components: [voice_assistant, speaker]
+    refresh: 0s
+  - source: github://esphome/esphome@jesserockz-2023-284-2
+    components: [esp_adf]
```

This can be reverted now
```suggestion
  - source: github://pr#5230
    components: esp_adf
```

---

ok, new commit incoming....